ttnn/ttnn/operations/unary.py

```diff
@@ -50,7 +50,7 @@ def _unary_validate_input_tensors(operation_name, input_tensor, *args, **kwargs)
             input_tensor,
             ranks=(2, 3, 4),
             dtypes=(ttnn.bfloat16, ttnn.bfloat8_b),
-            layouts=(ttnn.TILE_LAYOUT,),
+            layouts=(ttnn.TILE_LAYOUT, ttnn.ROW_MAJOR_LAYOUT),
```

Comment:
Is it necessary to allow ttnn.ROW_MAJOR_LAYOUT?

Reply:
> Is it necessary to allow ttnn.ROW_MAJOR_LAYOUT?

hey @eyonland,
Any reason why we should not allow ROW_MAJOR_LAYOUR? Perf reason?

Reply:
This is the requirement for this issue. https://github.com/tenstorrent-metal/tt-metal/issues/6184